integration-test/src/test/java/org/sagebionetworks/IT500SynapseJavaClient.java

```diff
@@ -1158,11 +1170,20 @@ public void testTeamAPI() throws SynapseException, IOException, InterruptedExcep
 
 		// query for team members using name fragment.  should get team creator back
 		String myDisplayName = myProfile.getUserName();
-		members = waitForTeamMembers(updatedTeam.getId(), myDisplayName, 1, 0);
+		members = waitForTeamMembers(updatedTeam.getId(), myDisplayName, TeamMemberTypeFilterOptions.ALL, 1, 0);
 		assertEquals(1L, synapseOne.countTeamMembers(updatedTeam.getId(), myDisplayName));
 		assertEquals(myPrincipalId, members.getResults().get(0).getMember().getOwnerId());
 		assertTrue(members.getResults().get(0).getIsAdmin());
-		
+
+		// query for team members using name fragment and admin status
```

Comment:
these cases should already be covered in the Autowired/unit tests at the manager level. Should only need to test for ability to send a request and get results back at this level